main.js

```diff
@@ -75,15 +75,20 @@ async function runEslint () {
   )
 
   const eslint = require(`${GITHUB_WORKSPACE}/node_modules/eslint`)
-  const cli = new eslint.CLIEngine()
+  const cli = new eslint.ESLint()
```

Comment:
Should we have some kind of error message for if a project is using an unsupported version of ESLint? Is there a sane way for us to output something to the user that says Balto can't run on this version?

Given the error handling in `run`, could it possibly be as simple as checking `eslint.version` and throwing an error if it's too low?

Reply:
Almost certainly. There _will_ be an error, but it would be meaningless to anyone not willing to go on an amazing journey of discovery. (A weird thing is that [`CLIEngine` still exists in the latest version](https://github.com/eslint/eslint/blob/v8.3.0/lib/cli-engine/index.js), but you definitely can't instantiate it, and I haven't looked into it enough to know why.)

Reply:
(If it wasn't clear, I'll look into throwing an error.)